app/errors.py

```diff
@@ -0,0 +1,18 @@
+class VaccineError(Exception):
+    """Base class for vaccine errors."""
+    pass
+
+
+class NotVaccinatedError(VaccineError):
+    """Raised when visitor has no vaccine."""
+    pass
+
+
+class OutdatedVaccineError(VaccineError):
+    """Raised when vaccine has been expired."""
+    pass
+
+
+class NotWearingMaskError(Exception):
+    """Raised when no mask on visitor."""
+    pass

```

Comment:
Consider adding a descriptive error message to the `NotWearingMaskError` exception. For example: `class NotWearingMaskError(Exception): def __init__(self, message='The person is not wearing a mask'): super().__init__(message)`.